crates/wdl-analysis/src/handlers/semantic_tokens.rs

```diff
@@ -285,7 +286,9 @@ fn resolve_identifier_ty(
     {
         return match name_info.ty() {
             Type::Call(_) => Some(SemanticTokenType::VARIABLE),
-            Type::Compound(CompoundType::Struct(_), _) => Some(SemanticTokenType::STRUCT),
+            Type::Compound(CompoundType::Custom(CustomType::Struct(_)), _) => {
+                Some(SemanticTokenType::STRUCT)
+            }
```

Comment:
Should we have a corresponding case for `CustomType::Enum` that returns `Some(SemanticTokenType::ENUM)`?

Reply:
Yup! I had added this this morning.